src/emulator/storage/files.ts

```diff
@@ -266,36 +287,26 @@ export class StorageLayer {
     return this._persistence.deleteAll();
   }
 
-  public finalizeUpload(uploadId: string): FinalizedUpload | undefined {
-    const upload = this._uploads.get(uploadId);
-
-    if (!upload) {
-      return undefined;
-    }
-
+  /**
+   * Stores the uploaded file with generated metadata and triggers CREATE Cloud Functions.
```

Comment:
should CREATE be FINALIZE?

Reply:
Yeah... I [think](https://cloud.google.com/functions/docs/calling/storage#event_types) it's `google.storage.object.finalize`, I'll put "Object Finalize".